Account/FxALoginStateMachine.swift

```diff
@@ -156,8 +156,8 @@ class FxALoginStateMachine {
                                 log.error("Unsuccessful HTTP request: \(error.description)!  Assuming error is transient and not transitioning.")
                                 return same
                             } else {
-                                log.error("Unknown error: \(error.description).  Transitioning to Separated.")
-                                return separated
+                                log.error("Unknown error: \(error.description).  Transitioning to Same.")
```

Comment:
As a very small nit, I think the log line above like "Assuming error is transient and not transitioning." is better than just "Transitioning to Same", as it also serves as a bit of an inline comment about why we're dong it.